twine-upload.sh

```diff
@@ -44,4 +44,4 @@ fi
 TWINE_USERNAME="$INPUT_USER" \
 TWINE_PASSWORD="$INPUT_PASSWORD" \
 TWINE_REPOSITORY_URL="$INPUT_REPOSITORY_URL" \
-  exec twine upload ${TWINE_EXTRA_ARGS} ${INPUT_PACKAGES_DIR%%/}/*
+  exec twine upload --verbose ${TWINE_EXTRA_ARGS} ${INPUT_PACKAGES_DIR%%/}/*
```

Comment:
This shouldn't be added unconditionally but it'd be acceptable to make it a configurable input on the action interface.

Reply:
ack. Working on it.